File: jquery/python/deleteCustomMetric.py
from google.cloud import monitoring

def delete_metric_descriptor(descriptor_name):
    # [START delete_metric_descriptor]
    client = monitoring.Client()

    descriptor = client.metric_descriptor(descriptor_name)
    descriptor.delete()

    print('Deleted metric descriptor {}.'.format(descriptor_name))

delete_metric_descriptor('projects/ird-cloud/metricDescriptors/custom.googleapis.com/my_metric')

File: jquery/nodejs/deleteCustomMetric.js
const monitoring = require('@google-cloud/monitoring');

const projectId = 'ird-cloud';
const metricId = 'custom.googleapis.com/stores/daily_sales';
const client = new monitoring.MetricServiceClient();

const request = {
	name : client.metricDescriptorPath(projectId,metricId),
}

client.deleteMetricDescriptor(request)
	.then(results => 
		{console.log(`Deleted ${metricId}`, results[0]);
	}).catch(err => {
		console.error("ERROR:", err);
	})
